js/workers/MeshLoader.js

```diff
@@ -2,42 +2,42 @@
  * Web Worker. Loads a mesh from STL file.
  */
 
-importScripts('../lib/three.min.js', '../lib/STLLoader.js');
 
-onmessage = function(e) {
-    var blob = e.data;
-    var reader = new FileReaderSync();
-    readContents(reader.readAsArrayBuffer(blob));
-};
+define([
+    'three'
+],
+function (STLLoader) {
```

Comment:
Here you are receiving the `THREE` object, but below we use the `STLLoader` object so you have to import both:

See this example with orbitcontrols:
https://github.com/biocore/emperor/blob/new-api/emperor/support_files/js/sceneplotview3d.js#L5

I think I may have misguided you!